envergo/moulinette/views.py

```diff
@@ -214,15 +214,19 @@ def get_results_url(self, form):
             )
 
         additional_forms = self.get_additional_forms(moulinette)
-        for form in additional_forms:
-            for field in form:
-                get.setlist(field.html_name, form.data.getlist(field.html_name))
+        for additional_form in additional_forms:
+            for field in additional_form:
+                get.setlist(
+                    field.html_name, additional_form.data.getlist(field.html_name)
+                )
```

Comment:
form est le nom de l'un des args de la méthode. Il était caché par la boucle for, ce qui rend le debugage désagréable.